CMakeLists.txt

```diff
@@ -23,7 +25,7 @@ set(RTRLIB_SRC rtrlib/rtr_mgr.c rtrlib/lib/utils.c rtrlib/lib/ip.c rtrlib/lib/ip
 set(RTRLIB_LINK ${RT_LIB} ${CMAKE_THREAD_LIBS_INIT})
 
 find_package(LibSSH 0.6.0)
-if(LIBSSH_FOUND)
+if(LIBSSH_FOUND AND NOT NOSSH)
```

Comment:
Nope, didn't tried that - i thought you mean some existing parameter in FindLibSSH.cmake. However, good point and better to have the SSH related stuff in one place. I moved the condition there, its still working. Thx @fho ...